node/index.js

```diff
@@ -3,7 +3,7 @@ var path = require('path');
 var amd = require('./amd-loader');
 
 var map = {
-  'p-promise': 'p-promise'
+  'promise': 'es6-promise'
```

Comment:
This part is causing issues at the moment. will need to adjust the node loader probably